source/administration/bucket-replication/enable-server-side-two-way-bucket-replication.rst

```diff
@@ -97,7 +97,7 @@ Considerations
    MinIO does *not* replicate delete operations resulting from the application of :ref:`lifecycle management expiration rules <minio-lifecycle-management-expiration>`. 
    Configure matching expiration rules on both the source and destination bucket to ensure consistent application of object expiration.
 
-   See :ref:`minio-replication-behavior-delete` for more complete documentation.
+   See :ref:`minio-replication-behavior-delete` and :ref:`minio-object-delete` for more complete documentation.
```

Comment:
An idea, while we are in here
```suggestion
   See :ref:`minio-replication-behavior-delete` and :ref:`minio-object-delete` for more information.
```

Reply:
We do seem to like the phrase `more complete documentation`, ok maybe leave it alone.

Reply:
67 uses across 35 files. We can talk about updating it, but probably not in this PR.